lib/charms/prometheus_k8s/v0/prometheus.py

```diff
@@ -241,6 +240,51 @@ def _on_scrape_targets_changed(self, event):
             prometheus_scrape_config.append(job)
         ...
 
+## Alerting Rules
+
+This charm library also supports gathering alerting rules from all
+related consumer charms and enabling corresponding alerts within the
+provider charm.  Alert rules are automatically gathered by Consumer
+charms when using this library, from a directory conventionally named
+`prometheus_alert_rules`. This directory must reside at the top level
+in the Consumer charm. Each file in this directory is assumed to be a
+single alert rule in YAML format. The format of this alert rule
+conforms to [Prometheus
+documentation](https://prometheus.io/docs/prometheus/latest/configuration/alerting_rules/).
+An example of the contents of one such file is shown below.
+
+```
+- alert: HighRequestLatency
+  expr: job:request_latency_seconds:mean5m{my_key=my_value, %%juju_topology%%} > 0.5
+  for: 10m
+  labels:
+    severity: Medium
+    type: HighLatency
+  annotations:
+    summary: High request latency for {{ $labels.instance }}.
+```
+
+It is **very important** to note the `%%juju_topology%%` filter in the
+expression for the alert rule shown above. This filter is a stub that
+is automatically replaced by the consumer charm's Juju topology. Such
```

Comment:
> the consumer charm's Juju topology

consider adding an example, e.g. "model-app-unitnum", because "juju topology" is vague

Reply:
Clarified.